packages/client/hmi-client/src/components/models/tera-model.vue

```diff
@@ -638,9 +615,12 @@ onUpdated(() => {
 	}
 });
 
+/*
+// apparently this is never used?
 const launchForecast = () => {
 	showForecastLauncher.value = true;
 };
+*/
```

Comment:
You removed the invocation above in the template section, so it is definitely no longer being used. 

Let's just remove it altogether instead of having the comment/code laying around.